### PR TITLE
Add option to allow saml version 2

### PR DESCRIPTION
**Pull request by jbuffin**
Fixes https://github.com/auth0/node-wsfed/issues/8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Options
 | signatureAlgorithm  | signature algorithm, options: rsa-sha1, rsa-sha256 | ```'rsa-sha256'``` |
 | digestAlgorithm     | digest algorithm, options: sha1, sha256          | ```'sha256'``` |
 | wctx                | state of the auth process                        | ```req.query.wctx``` |
+| samlVersion         | the version for the saml token `1|2`             | ```1``` |
```
Comment (siacomuzzi) [inline README.md]: `1.1|2.0`

File: lib/wsfed.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ var templates             = require('./templates');
 var PassportProfileMapper = require('./claims/PassportProfileMapper');
 var utils                 = require('./utils');
 var saml11 = require('saml').Saml11;
+var saml20 = require('saml').Saml20;
```
Comment (siacomuzzi) [inline lib/wsfed.js]: `var saml = require('saml');`

**Comment by siacomuzzi** [comment]
thanks for contributing! Could you please include some tests?

**Comment by jbuffin** [comment]
Sure will! I just wanted to get this in here to begin the conversation.

File: lib/wsfed.js [hunk 3]
```diff
@@ -76,7 +77,7 @@ module.exports = function(options) {
       var profileMap = options.profileMapper(user);
       var claims = profileMap.getClaims(options);
       var ni = profileMap.getNameIdentifier(options);
-      saml11.create({
+      (options.samlVersion === 2 ? saml20 : saml11).create({
```
Comment (siacomuzzi) [inline lib/wsfed.js]: please check if there are other available options for saml20.create

**Pull request closed without merging**